decidim-initiatives/app/views/decidim/initiatives/initiative_signatures/finish.html.erb

```diff
@@ -0,0 +1 @@
+OK!!!!
```

Comment:
This probably shouldn't be here 😄

Reply:
Ups!, sorry!, I'll remove this, the finish step is unused currently